dev/com.ibm.ws.microprofile.faulttolerance/src/com/ibm/ws/microprofile/faulttolerance/impl/ExecutorBuilderImpl.java

```diff
@@ -27,7 +27,7 @@
 import com.ibm.ws.threading.PolicyExecutorProvider;
 import com.ibm.wsspi.threadcontext.WSContextService;
 
-public class ExecutorBuilderImpl<T, R> implements ExecutorBuilder<T, R> {
+public class ExecutorBuilderImpl<R> implements ExecutorBuilder<R> {
```

Comment:
What is `R`?

Reply:
From the interface:
`* @param <R> the type of the result returned by {@code Executor}s created by this builder`

Reply:
ok